novu.code-workspace

```diff
@@ -110,7 +110,7 @@
     }
   ],
   "settings": {
-    "typescript.tsdk": "node_modules/typescript/lib",
+    "typescript.tsdk": "✨ novu root/node_modules/typescript/lib",
```

Comment:
what is that?

Reply:
I'll remove. I think vscode auto-added it when i used the ts version dropdown